src/Tickets/Commerce/Gateways/Stripe/Webhooks/Handler.php

```diff
@@ -103,4 +109,46 @@ public static function handle_default( WP_REST_Request $request, WP_REST_Respons
 	public static function handle_account_deauthorized( WP_REST_Request $request, WP_REST_Response $response ): WP_REST_Response {
 		return $response;
 	}
+
+	/**
+	 * Get the class and method to call to handle this event.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $type The event type from Stripe.
+	 *
+	 * @return string
+	 */
+	public static function get_handler_method( $type ) {
+		$parts  = explode( '.', $type );
+		$class  = implode( '_', array_map( 'ucwords', explode( '_', $parts[0] ) ) );
+		$method = $parts[1];
+
+		// If we have a Class::method named specific to the event, call it.
+		$name = __NAMESPACE__ . "\\{$class}::{$method}";
+
+		if ( ! function_exists( $name ) ) {
+			// If not, we fallback to the generic handle method available to all events.
+			$name = __NAMESPACE__ . "\\{$class}::handle";
+		}
+
+		return $name;
```

Comment:
Lets make this be a map of One-to-One, between the type and the callback, These dynamic resolutions can lead into bugs too easily.